File: springboot-scala/src/main/scala/com/example/scala/service/impl/UserServiceImpl.scala
package com.example.scala.service.impl

import com.baomidou.mybatisplus.extension.service.impl.ServiceImpl
import com.example.scala.dao.UserDao
import com.example.scala.model.User
import com.example.scala.service.IUserService
import org.springframework.stereotype.Service

@Service
class UserServiceImpl extends ServiceImpl[UserDao , User] with IUserService{

}

File: springboot-validated/src/main/java/com/example/validated/service/impl/UserServiceImpl.java
package com.example.validated.service.impl;

import com.baomidou.mybatisplus.extension.service.impl.ServiceImpl;

import com.example.validated.mapper.UserMapper;
import com.example.validated.model.User;
import com.example.validated.service.IUserService;
import org.springframework.stereotype.Service;

@Service
public class UserServiceImpl extends ServiceImpl<UserMapper, User> implements IUserService {

}
